### ISSUE TITLE
Model checkpoints, training data, and evaluation code

### ISSUE THREAD

**Issue by alvaro-budria** [MoyGcc/vid2avatar#58]
Hi, thanks for your work and for making the code available.

I want to compare my method against vid2avatar, and for that, I am trying to closely reproduce the experiments on the SynWild dataset.

When I run your code, despite obtaining qualitatively good results that are comparable to those in the paper, I can't seem to obtain as good Chamfer l2 and Normal Consistency (NC). For instance, on the 00070 SynWild scene, I am getting around 0.55 NC and 4 Chamfer l2.
This is probably due to how I normalize the GT mesh. Right now, I am generating a mesh inside the unit sphere, scaling it by the corresponding scaling factor, and then I run ICP to scale and align it with the GT Mesh. I am not sure though that this is what you did.

Thus I would like to kindly ask you to provide the model checkpoints and the evaluation scripts that you used, to ensure fair comparison and reproducibility.

Training data would be nice to share, but assuming you followed the exact same process as described in the readme in this repo, it's not too much of an issue.

Again, thanks for your work, the results are really nice.
